src/lnd-setup/generate-wallet-unlocker-client.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Wallet Unlocker Client Module
+ * @module src/lnd-setup/generate-wallet-unlocker-client
+ */
+
+const grpc = require('grpc')
+const loadProto = require('../utils/load-proto')
+const fs = require('fs')
+
+/**
+ * Generates a lnrpc.WalletUnlocker client which is only used on initialization of the LND
+ * node.
+ *
+ * @function
+ * @param {LndEngine} - engine
+ * @param {String} engine.host
+ * @param {String} engine.protoPath
+ * @param {String} engine.tlsCertPath
+ * @returns {grpc.Client} lnrpc WalletUnlocker client definition
+ * @returns {null}
+ */
+function generateWalletUnlockerClient ({ host, protoPath, tlsCertPath }) {
```

Comment:
this feels pretty similar to the other client - any way to refactor some of it (without losing legibility)?

Reply:
Since the implementation of `generate-wallet-unlocker` is pretty straight-forward, it would probably be more code to tear some of it out. I made the decision with this piece of code as a "times when it would be good to have duplication".

Having either the `WalletUnlocker` or `Lightning` services be returned from the same function, I found that all I was doing was adding in conditionals.

With this said, if you have suggestions for refactoring, let me know. I can re-evaluate this approach